lib/anonpay/anonpay_status_response.dart

```diff
@@ -0,0 +1,17 @@
+class AnonpayStatusResponse {
+  final String status;
+  final double? fiatAmount;
+  final String? fiatEquiv;
+  final double? amountTo;
```

Comment:
as far as I can see it's the only way to know the amount, and it should return the amount for both crypto and fiat.
as we discussed shouldn't this be the amount of fiat if `fiatEquiv` is not null, else it would be the crypto amount